#Operation State
uint8 state
